physics/satmedmfvdifq.meta

```diff
@@ -588,6 +588,13 @@
   dimensions = ()
   type = integer
   intent = in
+[tc_pbl]
+  standard_name = option_of_tc_application_in_boundary_layer_mass_flux_scheme
```

Comment:
Unless you envision tc_pbl being something other than 0/1, I would change this to a logical. Better yet, there already exists a variable in GFS_typedefs with the following metadata that could be used for this purpose:
`[hurr_pbl]`
 `standard_name = flag_hurricane_PBL`
 `long_name = flag for hurricane-specific options in PBL scheme`
 `units = flag`
 `dimensions = ()`
 `type = logical`

Reply:
@grantfirl, yes, in the future, this tc_pbl could potentially have options other than 0 and 1. The reason we set it as integer namelist option is that future choices of 2, 3, etc. can be added. Thanks!